main.js

```diff
@@ -0,0 +1,166 @@
+    details: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the  standard dummy text ever since the 1500s,when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries,',
+    technologies: ['Html', 'CSS', 'JavaScript', 'Ruby', 'bootsrap'],
+    live_link: 'www.google.com',
+    source_link: 'www.google.com',
+  },
+  {
+    id: 2,
+    name: 'Tonic',
+    tag: { sector: 'Canopy', type: 'Band End Dev', year: 2015 },
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: 'Assets/Snapshoot Portfolio Card3.svg',
+    details: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the  standard dummy text ever since the 1500s,when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries,',
+    technologies: ['Html', 'CSS', 'JavaScript', 'Ruby', 'bootsrap'],
+    live_link: 'www.google.com',
+    source_link: 'www.google.com',
+  },
+  {
+    id: 3,
+    name: 'Multi-Post Stories',
+    tag: { sector: 'Canopy', type: 'Band End Dev', year: 2015 },
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: 'Assets/Snapshoot Portfolio Card4.svg',
+    details: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the  standard dummy text ever since the 1500s,when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries,',
+    technologies: ['Html', 'CSS', 'JavaScript', 'Ruby', 'bootsrap'],
+    live_link: 'www.google.com',
+    source_link: 'www.google.com',
+  },
+];
+
+function insertTechList(data) {
+  let techList = '';
+  function insertTech(data) {
+    techList += `<li class="tech-langs">${data}</li>`;
+  }
+  data.technologies.map(insertTech);
+  return techList;
+}
+
+(function loadProjectData() {
+  let htmlSkeletonToInsert = '';
+  function insertData(data, index) {
+    const techList = insertTechList(data);
+    htmlSkeletonToInsert += `<div class="card">
+     <div class="card-img">
+       <img
+         src="${data.image}"
+         alt="Tonic-project-image"
+         title="Tonic-project"
+       />
+     </div>
+     <div class="card-content facebook-card">
+       <h3 class="card-title">${data.name}</h3>
+       <ul class="card-info">
+         <li class="bold-info">${data.tag.sector}</li>
+         <li class="info bullet"><span></span>${data.tag.type}</li>
+         <li class="info bullet"><span></span>${data.tag.year}</li>
+       </ul>
+       <p class="card-para">
+         ${data.description}
+       </p>
+       <ul class="card-lang">
+      ${techList}
+      </ul>
+       <button class="card-btn" data-value="${index}">See Project</button>
```

Comment:
- [ ] Kindly fix the spacing in these buttons.

- Your design
<img width="160" alt="image" src="https://github.com/siddghosh108/Portfolio-Popup-Window/assets/41163571/fd08319c-e27b-4c47-bcac-8753cd943f92">

- Figma design
<img width="192" alt="image" src="https://github.com/siddghosh108/Portfolio-Popup-Window/assets/41163571/a44bb008-6784-4e84-be27-40ff49019a4c">

And also kindly make the texts in both designs the same thing

Reply:
I cant see these buttons in my page. Please mention where you found them?

Reply:
Sorry i was mistaken, i thought you were mentioning something else, i Have increased the height spaging of these buttons now appearing in the popup window

![image](https://github.com/siddghosh108/Portfolio-Popup-Window/assets/131663359/aaf2d1f9-c038-499c-8a70-1b736588e306)